src/test/java/kg/jarkyn/cobspecserver/doubles/ExecutorServiceDouble.java

```diff
@@ -0,0 +1,90 @@
+package kg.jarkyn.cobspecserver.doubles;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.*;
+
+public class ExecutorServiceDouble implements ExecutorService {
```

Comment:
Is there a class that you could subclass instead? The interface is forcing you to implement a lot of methods you don't care about, I'd try subclassing and overriding just the important methods for your tests